draft-ietf-quic-v2.md

```diff
@@ -208,12 +208,35 @@ are willing to suffer a round-trip penalty if they are incorrect.
 # Applicability
 
 This version of QUIC provides no change from QUIC version 1 relating to the
-capabilities available to applications. Therefore, all Application Layer
+capabilities available to applications. Therefore, most Application Layer
 Protocol Negotiation (ALPN) ({{?RFC7301}}) codepoints specified to operate over
-QUICv1 can also operate over this version of QUIC.
+QUICv1 can also operate over this version of QUIC. Applications SHOULD NOT
+require new ALPNs for QUICv2 unless there is a specific performance or
+interoperability concern.
```

Comment:
I find this whole paragraph a bit confusing because of the different concepts getting munged together. V2 doesn't change how the TLS handshake works, so why are we saying that most of ALPN can work. 

In my mind it might be clear to state something like it is expected that applications can use QUIC v2 as a direct substitute for QUIC v1 but that there may be implications for how endpoints discover or agree upon the wire version in tandem with authenticated negotiation of an application protocol. For instance, an additional round trip if a client attempts v2 with a v1-only server. We could then go on to to talk about ALPN and rephrase the requirement to say that protocols that would like to avoid performance or interoperability concerns MAY define new ALPN identifiers that explicitly require QUICv2.